shared/ui/votingLink/votingLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { Link } from '@lidofinance/lido-ui';
+import { FC, PropsWithChildren, memo, useEffect, useState } from 'react';
+
+export const VotingLink: FC<PropsWithChildren<{ voteId?: string }>> = memo(
+  ({ voteId, children }) => {
+    const [href, setHref] = useState('https://vote.lido.fi');
+
+    useEffect(() => {
+      const url = new URL(window.location.href);
+      const tld =
+        url.hostname === 'localhost'
+          ? 'lido.fi'
+          : url.hostname.split('.').slice(-2).join('.');
+      const path =
+        voteId == null ||
+        voteId === '' ||
+        Number.isNaN(Number(voteId)) ||
+        voteId !== Math.trunc(Number(voteId)).toString()
+          ? ''
+          : `/vote/${voteId}`;
+      setHref(`https://vote.${tld}${path}`);
+    }, [voteId]);
+
+    return <Link href={href}>{children}</Link>;
```

Comment:
maybe just render nothing if no location is present...?

Reply:
Why? It's a bit better for SEO to have production value, and we don't really care if it's incorrect for testnet and stage

Reply:
sounds reasonable